### PR TITLE
Fixes: #151 created a book searching skill

### PR DESCRIPTION
**Pull request by isuruAb**
Fixes issue #151 
Changes: Added a book searching skill  and now it gives responses from Google Book API instead of web search.

Screenshots for the change: 
screen shot 2017-06-17 at 7 57 48 am

### CODE REVIEW

**Review by chashmeetsingh — APPROVED** [review]
Looks good.

**Review by madhavrathi — APPROVED** [review]

File: models/general/knowledge/en/books.txt [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#Returns the name of the book and link to read more
+what is this book * |is there any book called * ? | book * | is there a book called * ?
```
Comment (AliAyub007) [inline models/general/knowledge/en/books.txt]: Add more possible questions to it like. 
give me a book named *
Tell me about book * 
do you know any book called * 
can you give me a book called * 
can you provide me book * 
can you provide me book called  * 
what do you know about book * 
is there any book named * 

Think of more questions also.

**Review by saurabhjn76 — APPROVED** [review]
Looks good, please add more possible questions as @AliAyub007 suggested. :+1:

File: models/general/knowledge/en/books.txt (hunk 1 above)
Comment (isuruAb) [reply models/general/knowledge/en/books.txt]: nice i will add these questions

**Review by Laughing-Boy — APPROVED** [review]

**Pull request merged**